### PR TITLE
merge Rc0/v6.0.2

### PR DESCRIPTION
**Pull request by okwme**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -36,6 +36,11 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
 
+## [v6.0.2] - 2022-02-17
+
+* (gaia) Bump [Cosmos-SDK](https://github.com/cosmos/gaia/pull/1257) to [v0.44.6](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.44.3)
```
Comment (yaruwangway) [inline CHANGELOG.md]: the link of cosmos sdk is https://github.com/cosmos/cosmos-sdk

**Comment by yaruwangway** [comment]
since v6.0.1 is retracted, changelog v6.0.2 shall includes the changes of v6.0.1? if so, we also need to add `Bump [IBC](https://github.com/cosmos/ibc-go) module to [2.0.3](https://github.com/cosmos/ibc-go/releases/tag/v2.0.3).`

@okwme

File: CHANGELOG.md [hunk 2]
```diff
@@ -36,6 +36,11 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
 
+## [v6.0.2] - 2022-02-17
+
+* (gaia) Bump [Cosmos-SDK](https://github.com/cosmos/gaia/pull/1257) to [v0.44.6](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.44.6)
```
Comment (yaruwangway) [inline CHANGELOG.md]: * (gaia) Bump [Cosmos-SDK](https://github.com/cosmos/gaia/pull/1257) to [v0.44.6](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.44.6)
```suggestion
* (gaia) Bump [Cosmos-SDK](https://github.com/cosmos/cosmos-sdk) to [v0.44.6](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.44.6)
```

**Comment by okwme** [comment]
> since v6.0.1 is retracted, changelog v6.0.2 shall includes the changes of v6.0.1? if so, we also need to add `Bump [IBC](https://github.com/cosmos/ibc-go) module to [2.0.3](https://github.com/cosmos/ibc-go/releases/tag/v2.0.3).`
> 
> @okwme

I think that the changes can stay in v6.0.1, v6.0.2 is still built from v6.0.1, we just don't want golang to automatically download v6.0.1 for people. but people can still look in the changelog for v6.0.1. I'll make a special note in the release notes that since v6.0.1 is redacted that changes for both should be considered.

**Comment by okwme** [comment]
i've disabled LGTM since they don't recognize the `retract` directive (`/opt/src/go.mod:132: unknown directive: retract`).

**Pull request merged**